packages/editor/src/client/api/user.graphql

```diff
@@ -72,8 +71,8 @@ query User($id: ID!) {
     }
 }
 
-query SubscriptionAsCsv {
-  subscriptionAsCsv
+query SubscriptionsAsCsv {
+  csv: subscriptionsAsCsv
```

Comment:
why the renaming? Seems useless.

Reply:
I think it's good to have it like `subscriptionsCsvData.csv` instead, what do you think?